bbn-test-4/finality-providers/registry/Bido.json

```diff
@@ -0,0 +1,15 @@
+{
+  "description": {
+    "moniker": "Bido",
+    "identity": "",
+    "website": "https://app.bido.finance/",
+    "security_contact": "",
```

Comment:
Thanks! I would appreciate it if you could specify a contact and recreate the signature.

Reply:
@RafilxTenfen Hello, I have updated and re-signed.